### PR TITLE
Bug 1508790: Implement a provider for packet.net

### PR DESCRIPTION
**Pull request by walac**
Bugzilla Bug: [1508790](https://bugzilla.mozilla.org/show_bug.cgi?id=1508790)

### CODE REVIEW

File: services/worker-manager/package.json [hunk 1]
```diff
@@ -12,5 +12,8 @@
     "lint": "eslint src/*.js test/*.js",
     "fixlint": "eslint --fix src/*.js test/*.js",
     "test": "mocha test/*_test.js"
+  },
+  "dependencies": {
```
Comment (imbstack) [inline services/worker-manager/package.json]: All dependencies are at the top-level `package.json` now

File: services/worker-manager/schemas/v1/config-packet.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    minimum: 1
+    description: |
+      The number of tasks a single worker of this type
+      can run at any given time.
+  plan:
+    title: Plan
+    type: string
+    description: |
+      The plan to use to run this instance. This is equivalent
+      to the "instance type" in other cloud providers.
```
Comment (imbstack) [inline services/worker-manager/schemas/v1/config-packet.yml]: Thank you so, so, so much for this sort of comment.

File: services/worker-manager/package.json (hunk 1 above)
Comment (walac) [reply services/worker-manager/package.json]: Done!

**Comment by djmitche** [comment]
@walac I'm guessing we're not planning on doing this anymore?  Should we close this?

**Comment by walac** [comment]
Closed!

**Pull request closed without merging**